### PR TITLE
migration options

### LINKED ISSUE #543: Migration options paper
Options paper to outline migration routes with pros and cons. Should help inform comms strategy and timelines

### PR DESCRIPTION
**Pull request by acholyn**
## Resolves #543 

Don't have write access to tre-design so I've opened this here for feedback, not sure if too sensitive

cc: @lsheridanucl

Will loop Finley in closer to when he's back

### CODE REVIEW

File: docs/Portal-Migration-options.md [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+| **Benefits**           | - Agile: smaller chunks are more manageable for the Portal team<br>- Easier to respond to mishaps<br>- Fewer people affected if something goes wrong<br>- Allows decoupling of Studies and Projects, giving portal dev team more time to develop the DSH API and project data import functionality                                                                                                                                                 |
+| **Risks & Issues**     | - Users will have a period of using the Portal (researcher status) + SP (study management) + Share Manager/MyServices (project management) concurrently. This may be confusing<br>- Spread of/multiple comms to the same user groups<br>- May be long delay after Study phase to bring Project functionality to completion<br>- Using TRE users as the initial test group in each phase may be slightly slightly ineffective: it may test only trivial cases and exclude the complex edge cases where the true migration risk sits i.e. large and complex existing DSH Studies. |
```
Comment (jhughes982) [inline docs/Portal-Migration-options.md]: `slightly slightly` duplicate
